image_width: 1280
image_height: 720
camera_source: 0
camera_fps: 30
camera_name: camera
camera_matrix:
  rows: 3
  cols: 3
  data: [980.204294347984, 0.0, 671.1706506654324, 0.0, 980.6042943203598, 348.40211662988185,
    0.0, 0.0, 1.0]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.358410299696482, -0.007015055676821475, -0.006236952707830833, -0.007534344312732805,
    0.14749304865968124]
